decide/authentication/views.py

```diff
@@ -90,7 +90,7 @@ def post(self, request):
             encoded = base64.b64encode(
                 bytes(usernameToEncode, encoding="utf-8")
             ).decode("utf-8")
-            urlVerificar = f"{Site.objects.get_current().domain}/verificar/{encoded}"
+            urlVerificar = f"{request.META['HTTP_HOST']}/verificar/{encoded}"
```

Comment:
This is causing the tests to fail, please try to figure out why

Reply:
FelixoGudiel updated the code. Now all checks pass. Please check again.